### PR TITLE
Small changes to the SQL check operator concept guide

### PR DESCRIPTION
**Pull request by TJaniF**
Since this guide came out one bug has been fixed: it used to be necessary to separate aggregated from not-aggregated checks in the SQLTableCheckOperator, this is not necessary anymore.

Also both the SQLColumnCheckOperator and SQLTableCheckOperator will get a major update soon. This PR does not include the new functionality yet but moves datetime checks from the SQLColumn to the SQLTableCheckOperator because as of the next operator release datetime checks wont be possible anymore in the SQLColumnCheckOperator (min/max are only for numeric values).  

I am PRing this now because with the blog post coming out later this week I expect some traffic on the guide and think it should be up to date :)

### CODE REVIEW

**Review by kentdanas — APPROVED** [review]
LGTM!

File: learn/airflow-sql-data-quality.md [hunk 1]
```diff
@@ -27,8 +27,8 @@ The SQL Check operators are versions of the `SQLOperator` that abstract SQL quer
...
 
 - **[`SQLColumnCheckOperator`](https://registry.astronomer.io/providers/common-sql/modules/sqlcolumncheckoperator)**: Runs multiple predefined data quality checks on multiple columns within the same task.
-- **[`SQLTableCheckOperator`](https://registry.astronomer.io/providers/common-sql/modules/sqltablecheckoperator)**: Runs multiple checks involving aggregate functions for one or more columns.
-- **[`SQLCheckOperator`](https://registry.astronomer.io/providers/apache-airflow/modules/sqlcheckoperator)**: Takes any SQL query and returns a single row that is evaluated to booleans. This operator is useful for more complicated checks (e.g. including `WHERE` statements or spanning several tables of your database).
+- **[`SQLTableCheckOperator`](https://registry.astronomer.io/providers/common-sql/modules/sqltablecheckoperator)**: Runs multiple user-defined checks on one or more columns of a table.
+- **[`SQLCheckOperator`](https://registry.astronomer.io/providers/apache-airflow/modules/sqlcheckoperator)**: Takes any SQL query and returns a single row that is evaluated to booleans. This operator is useful for more complicated checks (e.g. spanning several tables of your database).
```
Comment (jwitz) [inline learn/airflow-sql-data-quality.md:31]:
```suggestion
- **[`SQLCheckOperator`](https://registry.astronomer.io/providers/apache-airflow/modules/sqlcheckoperator)**: Takes any SQL query and returns a single row that is evaluated to booleans. This operator is useful for more complicated checks that could span several tables of your database.
```

File: learn/airflow-sql-data-quality.md [hunk 2]
```diff
@@ -146,22 +146,20 @@ The following tests have failed:
...
 
-The `SQLTableCheckOperator` provides a way to check the validity of SQL statements containing aggregates over the whole table. There is no limit to the amount of columns these statements can involve or to their complexity. The statements are provided to the operator as a dictionary with the `checks` parameter.
+The `SQLTableCheckOperator` provides a way to check the validity of user defined SQL statements which can involve one or more columns of a table. There is no limit to the amount of columns these statements can involve or to their complexity. The statements are provided to the operator as a dictionary with the `checks` parameter.
 
 The `SQLTableCheckOperator` is useful for:
 
 - Checks that include aggregate values using the whole table (e.g. comparing the average of one column to the average of another using the SQL `AVG()` function).
 - Row count checks.
-- Schema checks.
+- Checking if a date is between certain bounds (e.g. using `MY_DATE_COL BETWEEN '2019-01-01' AND '2019-12-31'` to make sure only dates in the year 2019 occur).
```
Comment (jwitz) [inline learn/airflow-sql-data-quality.md:155]:
```suggestion
- Checking if a date is between certain bounds (for example, using `MY_DATE_COL BETWEEN '2019-01-01' AND '2019-12-31'` to make sure only dates in the year 2019 exist).
```

**Review by jwitz — APPROVED** [review]
Looks great!

**Pull request merged**